test/integration/analysis-controller/install.yaml

```diff
@@ -2,23 +2,19 @@ apiVersion: metrics.keptn.sh/v1alpha3
 kind: AnalysisValueTemplate
 metadata:
   name: ready
-  namespace: $NAMESPACE
 spec:
   provider:
     name: my-mocked-provider
-    namespace: $NAMESPACE
   query: 'sum(kube_pod_container_status_ready{namespace="{{.ns}}"})'
 ---
 apiVersion: metrics.keptn.sh/v1alpha3
 kind: AnalysisDefinition
 metadata:
   name: ed-my-proj-dev-svc1
-  namespace: $NAMESPACE
 spec:
   objectives:
     - analysisValueTemplateRef:
```

Comment:
it seems like we currently require the namespace to be set: https://github.com/keptn/lifecycle-toolkit/actions/runs/6219076090/job/16877728609?pr=2105#step:6:1499, so currently the integration test fails when this is removed. I'd say let's revert this change for now, i.e. re-add the `$NAMESPACE` and create a follow up ticket for making the namespace optional

Reply:
This should not be required, I assume this is a flaky run (as the other integration tests passed). I rebased the code and running it again, let's see